spec/features/merchant/invoices/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+require 'rails_helper'
```

Comment:
You're requiring 'rails_helper' twice here.

Reply:
Nice catch - just fixed